slu/tests/test_contextual_slu/data/input/nls-labels.yaml

```diff
@@ -0,0 +1,28 @@
+en_nls: &en_nls
+  NLS_LABEL_1: <prosody rate='90%'>For me to find your booking, please enter the 10-digit number used to make the booking, on your keypad. Press star key, if you don't know the number.</prosody>
```

Comment:
We don't have `state` in yaml files as we have in corresponding `csv`?

Reply:
Thanks for pointing this out.

In flow creator, `states `and `nls_labels `are defined & maintained separately ([example](https://console.skit.ai/65/flow-creator/#/clients/d6374943-5cb2-4a58-b6b1-ccbddf746fbc/flow/6694525b-3871-4a20-abf6-c424c173ea8c)). 
There should not be a state column in csv file too, because Studio is not supporting it. 

Right now,  `state `<-> `nls_label `mapping is not present in both the flow creator yaml and Studio csv files.

Reply:
Pushed an update